test/status/recipes/git.yaml

```diff
@@ -0,0 +1,19 @@
+root: true
+
+environment:
+    TestBranch: "master"
+
+depends:
+    - svn
+
+checkoutSCM:
+    - 
+      scm: git
+      url: https://github.com/BobBuildTool/bob.git
```

Comment:
The test setup should prepare a local git repository with some commits. That makes it more deterministic and it does not rely on the network.